src/Components/Common/DateInputV2.tsx

```diff
@@ -256,7 +256,7 @@ const DateInputV2: React.FC<Props> = ({
                   type="text"
                   readOnly
                   disabled={disabled}
-                  className={`cui-input-base cursor-pointer !px-2 disabled:cursor-not-allowed ${className}`}
+                  className={`cui-input-base cursor-pointer !px-2 pl-4 disabled:cursor-not-allowed ${className}`}
```

Comment:
`cui-input-base` class applies consistent padding and colors already. Removing `!px-2 pl-4` seems to have also solved the issue. Let's stick to not overriding the default padding applied for all input elements.

```suggestion
                  className={`cui-input-base cursor-pointer disabled:cursor-not-allowed ${className}`}
```